src/components/ticket-illustration/index.tsx

```diff
@@ -0,0 +1,18 @@
+import {useTheme} from '@atb/theme';
+import React from 'react';
+import * as Light from '@atb/assets/svg/color/illustrations/ticket-type/light';
+import * as Dark from '@atb/assets/svg/color/illustrations/ticket-type/dark';
+
+export type TicketIllustration = keyof typeof Light;
+
+const ThemedTicketIllustration = ({
+  name,
+}: {
+  name: TicketIllustration;
+}): JSX.Element => {
+  const {themeName} = useTheme();
+  const themeIcons = themeName === 'light' ? Light : Dark;
+  return themeIcons[name]({});
+};
```

Comment:
Kan være det er lurt å bruke det som React element fremfor funksjonskall. Enten med JSX eller f.eks React.createElement. F.eks noe som:

```ts
export type ThemedTicketIllustrationProps = { name: TicketIllustration } & SvgProps;
const ThemedTicketIllustration = ({
  name, ...props
}: ThemedTicketIllustrationProps): JSX.Element => {
  const {themeName} = useTheme();
  const themeIcons = themeName === 'light' ? Light : Dark;
  const Component = themeIcons[name]
  return <Component {...props} />
};
```

Da blir det mer navigerbart i debug stacks og React fungerer bedre i algoritmen sin for oppdateringer osv